packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -269,6 +273,18 @@ class WritingFlow extends Component {
 			return;
 		}
 
+		if ( metaKey || ctrlKey ) {
+			if ( ( metaKey && isUp ) || ( ctrlKey && keyCode === HOME ) ) {
```

Comment:
Is Ctrl+Home OS-specific? It doesn't do anything for me in other native inputs (this textarea on GitHub, a Google Docs document, a Sublime Text file), nor does any other combination of modifier + Home. In which case, we have the reverse problem that we're handling a combination we shouldn't be?

Note: While most standard Mac laptops won't have a physical Home key, in my case I connect by an [external keyboard](https://www.logitechg.com/en-us/products/gaming-keyboards/pro-gaming-keyboard.html) which does.